update_module_versions.py

```diff
@@ -4,11 +4,12 @@
 def get_module_versions():
     # gets the required module versions from `min_requirements.txt` file
     curdir = os.path.dirname(os.path.abspath(__file__))
-    req_file = os.path.join(
-        curdir, 'euphonic_sqw_models', 'min_requirements.txt')
-    if not os.path.isfile(req_file):
-        from update_dependencies import update_submodules
-        update_submodules('euphonic_sqw_models')
+    for submodule in [['light_python_wrapper', '+light_python_wrapper'],
```

Comment:
Urgh, `os.path`...

Reply:
Nothing wrong with that... I don't like to divide my paths...

Reply:
```python
>>> Path() / 0
ZeroDivisionError: division by zero
```
oh no!